cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -21,6 +21,7 @@ This section explains how to use Cypher to manage Neo4j role-based access contro
 *** <<administration-security-roles-show, Listing roles>>
 *** <<administration-security-roles-create, Creating roles>>
 *** <<administration-security-roles-drop, Deleting roles>>
+*** <<administration-security-roles-alter, Altering roles>>
```

Comment:
should it be named `Modifying roles` to be consistent with the user one (`Modifying users`)?